doc/source/reference/c-api/types-and-structures.rst

```diff
@@ -961,9 +959,14 @@ PyUFunc_Type and PyUFuncObject
 
        For each distinct core dimension, a set of ``UFUNC_CORE_DIM*`` flags
 
-       - :c:data:`UFUNC_CORE_DIM_CAN_IGNORE` if the dim name ends in ``?``
-       - :c:data:`UFUNC_CORE_DIM_SIZE_INFERRED` if the dim size will be
-         determined from the operands and not from a :ref:`frozen <frozen>` signature
+       .. c:macro:: UFUNC_CORE_DIM_IGNORE
+
+           if the dim name ends in ``?``
+
+       .. c:macro:: UFUNC_CORE_DIM_INFERRED
```

Comment:
In line 956 this is `UFUNC_CORE_DIM_SIZE_INFERRED`.

Reply:
Fixed.